source/common/orca/orca_load_metrics.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "envoy/upstream/host_description.h"
+
+#include "xds/data/orca/v3/orca_load_report.pb.h"
+
+namespace Envoy {
+namespace Orca {
+
+// List of metric names to report to the LRS.
+typedef std::vector<std::string> LrsReportMetricNames;
+
+void addOrcaLoadReportToLoadMetricStats(const OptRef<const LrsReportMetricNames>& metric_names,
```

Comment:
Now, the input parameter type should be `const  LrsReportMetricNames&` directly because it will never be nullopt.

And a minor tip: OptRef is something that similar to reference_wrapper/pointer which is a very light structure. You needn't use reference semantics for it.

Reply:
Done.